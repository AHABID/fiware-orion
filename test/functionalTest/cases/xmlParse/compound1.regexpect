1: +++++++++++++++++++++++++++++++++
HTTP/1.1 200 OK
Content-Length: 193
Content-Type: application/xml
Date: REGEX(.*)

<?xml version="1.0"?>
<updateContextResponse>
  <errorCode>
    <code>400</code>
    <reasonPhrase>Bad Request</reasonPhrase>
    <details>Unknown XML field: 'here'</details>
  </errorCode>
</updateContextResponse>
