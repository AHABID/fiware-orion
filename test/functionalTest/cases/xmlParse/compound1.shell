#   1. err: unknown path
#INVALID_XML
echo "1: +++++++++++++++++++++++++++++++++"
url='/ngsi10/updateContext'
payload='<?xml version="1.0"?>
<updateContextRequest>
  <contextElementList>
    <contextElement>
      <entityId type="Room" isPattern="false">
        <id>ConferenceRoom</id>
      </entityId>
      <contextAttributeList>
        <contextAttribute>
          <name>temperature</name>
          <type>
            <here>No compounds are allowed</here>
          </type>
          <contextValue>1</contextValue>
        </contextAttribute>
      </contextAttributeList>
    </contextElement>
  </contextElementList>
  <updateAction>APPEND</updateAction>
</updateContextRequest>'
curlXml ${url} "${payload}"

