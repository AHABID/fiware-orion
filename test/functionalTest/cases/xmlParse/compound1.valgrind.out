==26648== Memcheck, a memory error detector
==26648== Copyright (C) 2002-2011, and GNU GPL'd, by Julian Seward et al.
==26648== Using Valgrind-3.7.0 and LibVEX; rerun with -h for copyright info
==26648== Command: contextBroker -vvvvv -harakiri -port 9999 -pidpath /tmp/orion_9999.pid -db testharness -t 0-1
==26648== 
T@09:24:36  paConfigActions[727]: setting trace levels to '0-1'
E@09:24:36  exitFunc[559]: unlink(/tmp/orion_9999.pid): No such file or directory
==26648== 
==26648== HEAP SUMMARY:
==26648==     in use at exit: 17,690 bytes in 104 blocks
==26648==   total heap usage: 3,267 allocs, 3,163 frees, 281,093 bytes allocated
==26648== 
==26648== LEAK SUMMARY:
==26648==    definitely lost: 0 bytes in 0 blocks
==26648==    indirectly lost: 0 bytes in 0 blocks
==26648==      possibly lost: 0 bytes in 0 blocks
==26648==    still reachable: 17,690 bytes in 104 blocks
==26648==         suppressed: 0 bytes in 0 blocks
==26648== Rerun with --leak-check=full to see details of leaked memory
==26648== 
==26648== For counts of detected and suppressed errors, rerun with: -v
==26648== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 7 from 7)
F@09:24:36  ***[600]: Connected to mongo at localhost:testharness
==26653== Source and destination overlap in strncpy(0xa4d8a7, 0xa4d8a7, 63)
==26653==    at 0x4C29782: strncpy (mc_replace_strmem.c:463)
==26653==    by 0x659678: restInit(RestService*, IpVersion, char const*, unsigned short, char const*, char const*, void (*)(ConnectionInfo*), bool) (rest.cpp:600)
==26653==    by 0x5DF38D: main (contextBroker.cpp:737)
==26653== 
==26653== Source and destination overlap in strncpy(0xa4d8e2, 0xa4d8e2, 63)
==26653==    at 0x4C29782: strncpy (mc_replace_strmem.c:463)
==26653==    by 0x65969C: restInit(RestService*, IpVersion, char const*, unsigned short, char const*, char const*, void (*)(ConnectionInfo*), bool) (rest.cpp:603)
==26653==    by 0x5DF38D: main (contextBroker.cpp:737)
==26653== 
V@09:24:36  restStart[499]: Starting HTTP daemon on IPv4 0.0.0.0 port 9999
V@09:24:36  restStart[541]: Starting HTTP daemon on IPv6 :: port 9999
==26653== Thread 4:
==26653== Syscall param mmap(length) contains uninitialised byte(s)
==26653==    at 0x58B9AFA: mmap (syscall-template.S:82)
==26653==    by 0x4E398E7: MHD_pool_create (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35F47: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E39A1F: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35F20: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E39A24: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35F20: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E39A52: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35F20: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E399BC: MHD_pool_allocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E33C62: MHD_set_connection_value (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34033: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E3426D: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34783: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
2@09:24:44  xmlRequestGet[155]: Found xmlRequest of type 17, method 'POST' - index 12 (/updateContextRequest)
W@09:24:44  xmlParse[242]: ERROR: 'Unknown XML field: 'here'', PATH: '/updateContextRequest/contextElementList/contextElement/contextAttributeList/contextAttribute/type'   
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E399BC: MHD_pool_allocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E335B6: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E335BD: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4C2A639: memcpy (mc_replace_strmem.c:121)
==26653==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4C2A64E: memcpy (mc_replace_strmem.c:838)
==26653==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4C2A69B: memcpy (mc_replace_strmem.c:838)
==26653==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4C2A790: memcpy (mc_replace_strmem.c:838)
==26653==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Use of uninitialised value of size 8
==26653==    at 0x4C2A7C8: memcpy (mc_replace_strmem.c:838)
==26653==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Use of uninitialised value of size 8
==26653==    at 0x4C2A7ED: memcpy (mc_replace_strmem.c:838)
==26653==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Use of uninitialised value of size 8
==26653==    at 0x58CFC9C: __vsprintf_chk (vsprintf_chk.c:79)
==26653==    by 0x58CFC0E: __sprintf_chk (sprintf_chk.c:33)
==26653==    by 0x4E33622: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Use of uninitialised value of size 8
==26653==    at 0x4C2953C: strcpy (mc_replace_strmem.c:429)
==26653==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4C29569: strcpy (mc_replace_strmem.c:112)
==26653==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4C2956E: strcpy (mc_replace_strmem.c:121)
==26653==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4C2957F: strcpy (mc_replace_strmem.c:429)
==26653==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Use of uninitialised value of size 8
==26653==    at 0x4E3363E: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Syscall param sendto(msg) contains uninitialised byte(s)
==26653==    at 0x58BE0AC: send (send.c:32)
==26653==    by 0x4E36D69: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35963: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35A87: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E39A1F: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E33BA1: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35A9D: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E39A24: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E33BA1: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35A9D: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E39A40: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E33BA1: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35A9D: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E39A1F: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35D17: MHD_connection_handle_read (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37648: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E39A24: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35D17: MHD_connection_handle_read (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37648: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Conditional jump or move depends on uninitialised value(s)
==26653==    at 0x4E39A52: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E35D17: MHD_connection_handle_read (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37648: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
==26653== Thread 3:
==26653== Syscall param munmap(length) contains uninitialised byte(s)
==26653==    at 0x58B9B27: munmap (syscall-template.S:82)
==26653==    by 0x4E3996B: MHD_pool_destroy (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E37030: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x4E3881C: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26653==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26653==    by 0x58BD0EC: clone (clone.S:112)
==26653== 
E@09:24:44  orionExit[548]: Received a 'DIE' request
==26653== 
==26653== HEAP SUMMARY:
==26653==     in use at exit: 23,191 bytes in 148 blocks
==26653==   total heap usage: 3,611 allocs, 3,463 frees, 477,727 bytes allocated
==26653== 
==26653== LEAK SUMMARY:
==26653==    definitely lost: 0 bytes in 0 blocks
==26653==    indirectly lost: 0 bytes in 0 blocks
==26653==      possibly lost: 1,394 bytes in 16 blocks
==26653==    still reachable: 21,797 bytes in 132 blocks
==26653==         suppressed: 0 bytes in 0 blocks
==26653== Rerun with --leak-check=full to see details of leaked memory
==26653== 
==26653== For counts of detected and suppressed errors, rerun with: -v
==26653== Use --track-origins=yes to see where uninitialised values come from
==26653== ERROR SUMMARY: 42 errors from 29 contexts (suppressed: 7 from 7)
